scripts/newinstall.sh

```diff
@@ -282,11 +306,18 @@ set -e
 ##########	Download optional component (python, git, ...)
 
 if true; then
-	if [[ $WITH_MINICONDA2 = 1 ]]; then
```

Comment:
serves me right for doing this straight after some Python code. Will fix. `[[ ]]` is what has been in this file for a while now.